### DISCUSSION TITLE
Is there a way to configure freshrss to use client TLS certificate when updating feeds?

### DISCUSSION THREAD

**Discussion by joeydumont** [FreshRSS/FreshRSS#6978] [Q&A] [answered]
I want to use FreshRSS in a corporate environment, although mostly for myself, to read internal RSS feeds. However, some sites require TLS client authentication using a certificate. Is there a way to configure freshrss to use my client TLS certificate when pulling feeds?

It seems that freshrss uses libcurl in to query the feeds, but I couldn't find any environment variable that would configure client certificates, only CLI options for the curl tool. 

Thanks

**Comment by Alkarex** [comment, marked as answer]
Hello,
The way to go would be to help finalise the draft of PR https://github.com/FreshRSS/FreshRSS/pull/4573 to expose more per-feed cURL options. See issue https://github.com/FreshRSS/FreshRSS/issues/3752

Meanwhile, one option is to use the global curl options in `./data/config.php` on the model of:
https://github.com/FreshRSS/FreshRSS/blob/e9ce099277bd5cb0257b9535cfaed42d87670b88/config.default.php#L130-L143

**Reply by joeydumont** [reply]
Thank you, that worked perfectly!
